frontend/src/main/java/com/softgroup/configuration/RestApplicationConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.softgroup.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.FilterType;
+
+/**
+ * Created by alex on 16.02.17.
+ */
+
+@Configuration
+@ComponentScan(basePackages = "com.softgroup",
```

Comment:
@Alexey007gold this configuration should scan only this package. Other packages must have their own configurations.